src/components/CIPipelineN/CustomImageTags.tsx

```diff
@@ -270,7 +270,6 @@ function CustomImageTags({
                     </div>
                     <div className="" style={{ width: '32px', height: '20px' }}>
                         <Toggle
-                            disabled={window._env_.FORCE_SECURITY_SCANNING && formData.enableCustomTag}
```

Comment:
@shivani170 @vivek-devtron Why this check is removed? Will it not break the FORCE_SECURITY_SCANNING capability?

Reply:
@prakarsh-dt